ci/travis_script.sh

```diff
@@ -60,5 +64,13 @@ if [ "${PYSMT_SOLVER}" == "all" ];
 then
     python install.py --msat --conf --force;
     cp -v $(find ~/.smt_solvers/ -name mathsat -type f) /tmp/mathsat;
-    (for ex in `ls examples/*.py`; do echo $ex; python $ex || exit $?; done);
+
+    # since we're relying on relative `pysmt` import in examples,
+    # ensure `.` is added to `sys.path`
+    export PYTHONPATH=${PYTHONPATH:-.}
+
+    for ex in examples/*.py; do
+        echo $ex
+        python $ex
```

Comment:
This does not have the `exit` anymore. If the example fails, will the script terminate with an error code? If this is not the case, Travis will not know that the test failed.

Reply:
Doesn't Travis use something like `set -x`, so the script/test terminates on the first failed command?

This loop should be functionally identical to the one you had -- where you had `exit`, but from a separate subshell. This: `(python $ex; exit $?)` is identical to `python $ex` (in terms of exit status).

Reply:
Just to clarify, my motivation for refactoring this loop was the use of [`for in ls results` antipattern](http://mywiki.wooledge.org/ParsingLs). I replaced subshell invocations as well because I don't see the need for them here. Correct me if I'm wrong.